src/routes.js

```diff
@@ -0,0 +1,21 @@
+const Logger = require('./logger');
+const RootController = require('./controllers/root');
+
+const routes = {
+    '/': { get: RootController.get },
+}
+
+function setupAllRoutesInApp(app, routes) {
```

Comment:
We could go a different way (I personally prefer but that's up to you):

In `src/routes.js`:
```js
module.exports = (app) => {
  app.get("/", app.get("controllers").root.get);
}
```

In `server.js`: 
```js
const app = express();
const controllers = require("./controllers");

app.set("controllers") = controllers;

require("./routes")(app);
```

In `./controllers/index.js`:
```js
module.exports = {
  root: require("./root")
}
```

Reply:
I specially do not want to take the `app` object by injection, as it's a express server object

Reply:
Je veux que `routes.js` ne dépende pas de l'app express et expose un simple objet. Objectif: pouvoir sortir l'objet et sa doc dans un endpoint.